DllImportGenerator/DllImportGenerator/DllImportGenerator.cs

```diff
@@ -161,7 +161,7 @@ private static bool IsGeneratedDllImportAttribute(AttributeSyntax attrSyntaxMayb
 
                 // Found the GeneratedDllImport, but it has an error so report the error.
                 // This is most likely an issue with targeting an incorrect TFM.
-                if (attrData.AttributeClass.TypeKind == TypeKind.Error)
+                if (attrData.AttributeClass?.TypeKind is null or TypeKind.Error)
```

Comment:
I don't see how `AttributeClass` can ever be `null`. Perhaps in this case it makes more sense to `!`

Reply:
So I just confirmed with Fred that AttributeClass can be null in the case of errors when decoding from metadata.